db/seeds.d/100-installation_media.rb

```diff
@@ -6,17 +6,7 @@
 Medium.without_auditing do
   [
     {
-      :name => "CentOS 7 mirror",
-      :os_family => "Redhat",
-      :path => "http://mirror.centos.org/centos/$major/os/$arch",
```

Comment:
This old CentOS mirror is still available for older CentOS versions, so can we keep both `CentOS Stream` and `CentOS` media's?

Reply:
It's not: mirror.centos.org no longer has any DNS records and the whole infrastructure has been retired. And I don't want to set something up pointing to vault.centos.org.

Reply:
Fair enough, upon checking, I discovered 8-stream has only a readme which states "This directory (and version of CentOS) is deprecated.". So, we're fine with the current approach since 8-stream is already EOL